docs/volcano_integration.md

```diff
@@ -0,0 +1,107 @@
+# Integration with Volcano for Batch Scheduling
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workloads.
+With the integration with Volcano, Flink job and task managers can be scheduled for better scheduling efficiency.
```

Comment:
For average users, they might want to learn what specific problems are solved by the integration, then they can better judge whether it is a good fit for their use case.

Reply:
make sense